ui/package.json

```diff
@@ -51,7 +52,7 @@
       "footer-max-length": [
         2,
         "always",
-        72
+        100
```

Comment:
What was this for?

Reply:
To allow the `Contributes-to:` to work (seems like the last two lines counted as the footer) - happy to remove however as I've not got any proof as to "why" it works

Reply:
If it allows `Contributes-to:` I have no objections!